src/components/WorkspaceDashboardTaskRunsCard.vue

```diff
@@ -113,15 +113,57 @@
     }
   })
 
+  const maxValue = computed(() => {
+    const completedValues = taskRunsChartData.value.completed.map(([, y]) => y)
+    const failedValues = taskRunsChartData.value.failed.map(([, y]) => y)
+    const minValue = 1
+    const max = Math.max(...completedValues, ...failedValues, minValue)
+
+    return max
+  })
+
+  const maxCompletedValue = computed(() => {
+    const completedValues = taskRunsChartData.value.completed.map(([, y]) => y)
+    const minValue = 1
+    const maxCompleted = Math.max(...completedValues, minValue)
+
+    if (maxCompleted === maxValue.value) {
+      return maxCompleted
+    }
+
+    let unit = 1
+
+    while (unit <= 10) {
+      if (maxCompleted > maxValue.value / unit) {
+        return maxCompleted * unit
+      }
+
+      unit++
+    }
+
+    return maxCompleted * unit
+  })
+
   const maxFailedValue = computed(() => {
-    const values = taskRunsChartData.value.failed.map(([, y]) => y)
-    const max = Math.max(...values)
+    const failedValues = taskRunsChartData.value.failed.map(([, y]) => y)
+    const minValue = 1
+    const maxFailed = Math.max(...failedValues, minValue)
+
+    if (maxFailed === maxValue.value) {
+      return maxFailed
+    }
+
+    let unit = 1
+
+    while (unit <= 10) {
```

Comment:
Why 10? (here and in the completed computed)

Reply:
Just a nice round number lol. I should make that a const. These basically just reduce the distance between the max completed value and the max failed value so you get at least some movement in the lines. Otherwise a large complete number completely flattens a tiny failed number.